### DISCUSSION TITLE
convert Celsius to Fahrenheit?

### DISCUSSION THREAD

**Discussion by lumpynose** [Koenkk/zigbee2mqtt#11526] [General]
I have a SONOFF  SNZB-02 and it returns the temperature in Celsius.  Is there some place in the zigbee2mqtt code where I could add some conversion code to convert it to Fahrenheit?

**Comment by eigenphase** [comment]
Any solution to this?

I have the same issue with my Centralite thermostat: The device is set to fahrenheit, the device specific settings are explicitely set to fahrenheit, configuration.yaml explicitely has `thermostat_unit: fahrenheit` set, yet z2m still shows it with Celsius!
